packages/playground/css/imported.css

```diff
@@ -20,5 +20,5 @@ pre {
 
 /* test url comma separation */
 .url-separated {
-  background-image:url(images/cat.webp),url(images/dog.webp);
-}
\ No newline at end of file
+  background-image: url(images/cat.webp), url(images/dog.webp);
```

Comment:
This one should be ignored, If I recall correctly the issue appeared when there wasn't a space